packages/react-components/src/components/MeetingConferencePhoneInfo.tsx

```diff
@@ -0,0 +1,260 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+import React, { useCallback } from 'react';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { useMemo } from 'react';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { IModalStyles, Modal, Stack, useTheme, Text, IconButton, Icon } from '@fluentui/react';
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+import {
+  themedPhoneInfoModalStyle,
+  titleClassName,
+  titleContainerClassName,
+  phoneInfoTextStyle,
+  phoneInfoIcon,
+  phoneInfoInctructionLine,
+  phoneInfoStep,
+  stepTextStyle,
+  infoConnectionLinkStyle
+} from './styles/TeamsMeetingConferenceInfo';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { _preventDismissOnEvent } from '@internal/acs-ui-common';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { useLocale } from '../localization';
+/* @conditional-compile-remove(teams-meeting-conference) */
+import { isPossiblePhoneNumber, parsePhoneNumber } from 'libphonenumber-js';
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @internal
+ * Information for conference phone info
+ */
+export interface _ConferencePhoneInfo {
+  phoneNumber: string;
+  conferenceId: string;
+  isTollFree: boolean;
+  country?: string;
+  city?: string;
+}
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * strings for phone info modal
+ * @beta
+ */
+export interface MeetingConferencePhoneInfoModalStrings {
+  /**
+   * Header for the phone info modal
+   */
+  meetingConferencePhoneInfoModalTitle?: string;
+  /**
+   * Phone number instruction
+   */
+  meetingConferencePhoneInfoModalDialIn?: string;
+  /**
+   * Meeting ID instruction
+   */
+  meetingConferencePhoneInfoModalMeetingId?: string;
+  /**
+   * Wait for phone connection
+   */
+  meetingConferencePhoneInfoModalWait?: string;
+  /**
+   * Tool Free Phone Label
+   */
+  meetingConferencePhoneInfoModalTollFree?: string;
+  /**
+   * Tool Phone Label
+   */
+  meetingConferencePhoneInfoModalToll?: string;
+  /**
+   * No phone number available message
+   */
+  meetingConferencePhoneInfoModalNoPhoneAvailable?: string;
+}
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @internal
+ * MeetingConferencePhoneInfoModal Component Props.
+ */
+export interface _MeetingConferencePhoneInfoModalProps {
+  conferencePhoneInfoList: _ConferencePhoneInfo[];
+  showModal?: boolean;
+  onDismissMeetingPhoneInfoSettings?: () => void;
+}
+
+/* @conditional-compile-remove(teams-meeting-conference) */
+/**
+ * @internal
+ * a component for setting spoken languages
+ */
+export const _MeetingConferencePhoneInfoModal = (props: _MeetingConferencePhoneInfoModalProps): JSX.Element => {
```

Comment:
This does seem like a pretty good component to me that could be exported for Contoso to use. I wonder if this is a feature we are wanting to put behind the Composite wall? or do we think there is Value here to add this component to the package?

Reply:
made public